cli/running/base_instance.go

```diff
@@ -110,8 +110,8 @@ func StdErrOpt(writer io.Writer) InstanceOption {
 func StdLoggerOpt(logger ttlog.Logger) InstanceOption {
 	return func(inst *baseInstance) error {
 		inst.logger = logger
-		inst.stdOut = logger.Writer()
-		inst.stdErr = logger.Writer()
+		inst.stdOut = logger
+		inst.stdErr = logger
```

Comment:
https://github.com/tarantool/tt/blob/da042d0ad022712948851a7bf2acfc8f9561b35f/cli/running/running.go#L761-L762

But stdOut may be overwritten.